packages/shared/prisma/migrations/20251002153814_add_backfill_billing_cycle_anchors_background_migration/migration.sql

```diff
@@ -0,0 +1,2 @@
+INSERT INTO background_migrations (id, name, script, args)
+VALUES ('2025-0102-backfill-billing-cycle-anchors', '20250102_backfill_billing_cycle_anchors', '20250102_backfillBillingCycleAnchors', '{}');
```

Comment:
We usually use a UUID for the id and there is no need to have the date in the script name.

Reply:
updated